data.sql

```diff
@@ -6,9 +6,30 @@ INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALU
 
 --PROJECT TWO
 INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Charmander','2020-02-08',false,-11,0);
-INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Plantmon','2021-11-15',true,-5.7,2);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempts) VALUES ('Plantmon',);
```

Comment:
- [ ] You removed the remaining data for `Plantmon` when inserting this record in the `animals` table. Kindly reverse it and insert the full data for this record. 👍